v4/source/users.yaml

```diff
@@ -1272,6 +1652,30 @@
           $ref: '#/responses/NotFound'
         '501':
           $ref: '#/responses/NotImplemented'
+      x-code-samples:
+        - lang: 'Go'
+          source: |
+            import "github.com/mattermost/mattermost-server/model"
+
+            Client := model.NewAPIv4Client("https://your-mattermost-url.com")
+            Client.Login("email@domain.com", "Password1")
+
+            currentService := "email"
+            newService := "gitlab"
+            email := "test@domain.com"
+            password := "awesomePassword"
+            mfaCode := "adWv1qPZmHdtxk7Lmqh6RtxWxS"
+            ldapLoginID := "RdDjEDlkWgt7ndjyVLwWGvnX8c"
+
+
+            link, resp := Client.SwitchAccountType(&model.SwitchRequest{
```

Comment:
This is inconsistent with how you did this before. Would you please fix this?

Reply:
done